src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+                    }
+                    break;
+
+                case "mark":
+                    try {
+                        int item = Naruto.string2Int(input[1]);
+                        System.out.println(naruto.markItem(item - 1));
+                    } catch(NumberFormatException e) {
+                        System.out.println("Whoops! it seems you your index is not an integer Dattebayo!" +
+                                "\n'mark <Index>'");
+                    } catch(ArrayIndexOutOfBoundsException e) {
+                        System.out.println("Whoops! mark needs the index of the item Dattebayo!" +
+                                "\n'mark <Index>'");
+                    }
+                    break;
+
+                case "unmark":
+                    try {
+                        int index = Naruto.string2Int(input[1]);
+                        System.out.println(naruto.unMarkItem(index - 1));
+                    } catch(NumberFormatException e) {
+                        System.out.println("Whoops! it seems you your index is not an integer Dattebayo!" +
+                                "\n'unmark <Index>'");
+                    } catch(ArrayIndexOutOfBoundsException e) {
+                        System.out.println("Whoops! unmark needs the index of the item Dattebayo!" +
+                                "\n'unmark <Index>'");
+                    }
+                    break;
+
+                case "delete":
+                    try {
+                        int index = Naruto.string2Int(input[1]);
+                        System.out.println(naruto.deleteItem(index - 1));
+                    } catch(NumberFormatException e) {
+                        System.out.println("Whoops! it seems you your index is not an integer Dattebayo!" +
+                                "\n'delete <Index>'");
+                    } catch(ArrayIndexOutOfBoundsException e) {
+                        System.out.println("Whoops! delete needs the index of the item Dattebayo!" +
+                                "\n'delete <Index>'");
+                    }
+                    break;
+
+                case "bye":
+                    System.out.println(naruto.bye());
+                    breakLoop = true;
+                    break;
+
+                case "help":
+                    System.out.println("    Hi are you in need of help? These are the commands you can use Dattebayo:" +
+                            "\n      todo <Task>: add a task you need to do" +
+                            "\n      deadline <Task> /by<Deadline>: add a deadline you need to complete" +
+                            "\n      event <Task> /at<Time>: add an event you need to attend" +
+                            "\n      mark <index>: mark an task as completed by index" +
+                            "\n      unmark <index>: unmark a completed task by index" +
+                            "\n      list: view all you tasks by index" +
```

Comment:
Maybe it would be neater to do the line break and place the "+" symbol at the start of the next line instead of the end of previous line. Similar cases spotted above.